pkg/controller/garbagecollector/garbagecollector.go

```diff
@@ -684,10 +684,21 @@ func (gc *GarbageCollector) processItem(item *node) error {
 }
 
 func (gc *GarbageCollector) Run(workers int, stopCh <-chan struct{}) {
+	glog.Infof("The garbage collector is starting to run")
 	for _, monitor := range gc.monitors {
 		go monitor.controller.Run(stopCh)
 	}
 
+	wait.PollInfinite(10*time.Second, func() (bool, error) {
+		for _, monitor := range gc.monitors {
+			if !monitor.controller.HasSynced() {
```

Comment:
Might as well log here, too. "Garbage Collector: Waiting for resource listings to finish..."